extensions/notification/NotificationConfigManager.js

```diff
@@ -1,19 +1,73 @@
 const joi = require('joi');
 
+const { ZenkoMetrics } = require('arsenal').metrics;
 const LRUCache = require('arsenal').algorithms
     .cache.LRUCache;
 const MongoClient = require('mongodb').MongoClient;
-
 const constants = require('./constants');
 
 const paramsJoi = joi.object({
     mongoConfig: joi.object().required(),
+    origin: joi.string().required(),
     logger: joi.object().required(),
 }).required();
 
 const MAX_CACHED_ENTRIES = Number(process.env.MAX_CACHED_BUCKET_NOTIFICATION_CONFIGS)
     || 1000;
 
+// Config manager is used in both the QueuePopulator and QueueProcessor
+// we use this label to distinguish between the two
+const CONFIG_MANAGER_LABEL_ORIGIN =  'origin';
```

Comment:
do we actually need this?
We will already be able to differentiate by looking at the job/service/container labels (which are added by prometheus)